app/controllers/feedbacks_controller.rb

```diff
@@ -93,8 +93,8 @@ class FeedbacksController < ApplicationController
   end
 
   post '/feedbacks/delete/:id' do
```

Comment:
this route should be `delete '/feedbacks/:id'` make sure that your form has a hidden input field with a `value="DELETE` and a `name="_method"`